check_nextcloud.php

```diff
@@ -22,21 +23,75 @@ function convert_filesize($bytes, $decimals = 2) {
   return sprintf("%.{$decimals}f %s", $bytes / pow(1024, $factor), @$size[$factor]);
 }
 
+function check_range($pattern,$value) {
+    if(preg_match('/@([-0-9\.]+)\:([-0-9\.]+)/',$pattern,$matches)) {
+        return ( $value >= $matches[1] and $value <= $matches[2] )? 1 : 0;
+    }
+    elseif(preg_match('/([-0-9\.]+)\:([-0-9\.]+)/',$pattern,$matches)) {
+        return ( $value < $matches[1] or $value > $matches[2] )? 1 : 0;
+    }
+    elseif(preg_match('/\~\:([-0-9\.]+)/',$pattern,$matches)) {
+        return ( $value > $matches[1])? 1 : 0;
+    }
+    elseif(preg_match('/([-0-9\.]+)\:$/',$pattern,$matches)) {
+        return ( $value < $matches[1] ) ? 1 : 0;
+    }
+    elseif(preg_match('/([-0-9\.]+)/',$pattern,$matches)) {
+        return ($value < 0 or $value > $matches[1])?1:0;
+    }
+    else {
+        return 0;
+    }
+}       
+
 function show_helptext() {
   print "check_nextcloud.php - Monitoring plugin to check the status of nextcloud serverinfo app.\n
 You need to specify the following parameters:
-  -H:  hostname of the nextcloud instance, e.g. cloud.example.com
+  -H hostname: domain address of the nextcloud instance, e.g. cloud.example.com
```

Comment:
Removing the `:` is good I think, but please keep the spaces between the parameter and the description, it makes it much more readable.

Reply:
ok